docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -12,7 +12,9 @@ This page introduces additional configurations for Intel® Processor Graphics (G
 
 ## Linux
 
-If you have installed OpenVINO Runtime via the installer, APT, or YUM, follow these steps to work with GPU:
+If you have installed OpenVINO Runtime via the installer, APT, or YUM, follow these steps to work with GPU. If you installed OpenVINO™ Runtime using PyPI or Anaconda Cloud, scripts are not installed locally and must be accessed through GitHub.
+
+Source code for the instalation scripts is also available in the [OpenVINO™ scripts repository on GitHub](https://github.com/openvinotoolkit/openvino/tree/master/scripts).
```

Comment:
we need to have better instructions, see CVS-79326
CC @artanokhov

Reply:
Thanks @ilya-lavrenov, wasn't aware about it. Should this PR be closed in favor of 79326?

Reply:
Suppose yes